src/Microsoft.AspNetCore.Routing/Resources.resx

```diff
@@ -130,7 +130,7 @@
     <value>The supplied route name '{0}' is ambiguous and matched more than one route.</value>
   </data>
   <data name="DefaultHandler_MustBeSet" xml:space="preserve">
-    <value>A default handler must be set on the RouteCollection.</value>
+    <value>A default handler must be set on the RouteBuilder.</value>
```

Comment:
Could you pass this instead? Type names can be passed in to since they don't get localized.